### ISSUE TITLE
Command messenger:monitor:purge reduced options

### ISSUE THREAD

**Issue by golliholzland** [zenstruck/messenger-monitor-bundle#38]
It would be nice to accept customized ranges such ass 2 weeks / 3 weeks ....

**Comment by kbond** [comment]
`messenger:monitor:purge --older-than="2 weeks ago"` doesn't work?

**Comment by golliholzland** [comment]
> `messenger:monitor:purge --older-than="2 weeks ago"` doesn't work?

No, we are limited to the defined ENUM  
![image](https://github.com/zenstruck/messenger-monitor-bundle/assets/132348430/487580a2-27fe-4a9d-8dcf-cf47c742653b)

**Comment by kbond** [comment]
Got it, yeah, I'll need to refactor this to allow what you describe.

**Comment by golliholzland** [comment]
**I just had another Idea :** 
it would be beneficial to include a "keep" option. This "keep" option should accept an integer value, and its purpose should be to specify how many of the most recent messages should be retained for each type of transport. Additionally, it would be helpful if we could customize this retention setting for each type of transport.

**Comment by kbond** [comment]
> it would be beneficial to include a "keep" option

Yes, that would line up with the behaviour of the `messenger:monitor:schedule:purge`.

**Comment by golliholzland** [comment]
> > it would be beneficial to include a "keep" option
> 
> Yes, that would line up with the behaviour of the `messenger:monitor:schedule:purge`.

I tried this but it didn't work, I check it again.
Thanks for the fast answer
